docs/specs/toc.yml

```diff
@@ -914,6 +914,7 @@ inputs:
     - name: Invalid TocHref Reference
       tocHref: index.md
 outputs:
+  docs/toc.json:
```

Comment:
This diff is a quirk. Before the change, the error of included file is marked on the includer [here](https://github.com/dotnet/docfx/pull/5621/files#diff-9a6c1c1d834540f46bd38fdef1f759adL166), now the logic is more consistent: exclude files that originate the error.

Reply:
From the test title, the error should not be assigned to its parent, but seems that it did before the change, now it's better

Reply:
yep